plugins/obs-ffmpeg/obs-ffmpeg-output.c

```diff
@@ -357,20 +363,25 @@ static bool create_audio_stream(struct ffmpeg_data *data)
 	if (data->output->oformat->flags & AVFMT_GLOBALHEADER)
 		context->flags |= CODEC_FLAG_GLOBAL_H;
 
-	return open_audio_codec(data);
+	return open_audio_codec(data, idx);
 }
 
 static inline bool init_streams(struct ffmpeg_data *data)
 {
+	int i;
```

Comment:
Is there a need to initialize this loop variable way up here instead of in the `for` loop init step?

Reply:
nope; just following ffmpeg c style; mixing up projects and styles , sorry